Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSupportCards.java

```diff
@@ -1094,6 +659,13 @@ public static String findDirectDownloadKey(String setCode, String cardName, Stri
         return null;
     }
 
+    public static String prepareCardNumber(String xmageCardNumber) {
+		if (xmageCardNumber.endsWith("*")) return xmageCardNumber.substring(0 , xmageCardNumber.length() -1) +"★" ;
```

Comment:
same style comments as above, also use spaces not tabs. Add doc comment on usage (e.g. replace non-ASCII characters in collector numbers for scryfall)